Platform/Sophgo/SG2042Pkg/About_Sophgo_platform.md

```diff
@@ -0,0 +1,40 @@
+# Sophogo SG2042 EDK2 RISC-V Platform Project    
+## Sophgo SG2042 Platform
+The development work for this project was carried out around the Sophgo SG2042 EVB development board. 
+The deployment of EDK2 was completed on the EVB test version. Below is the external appearance display of the EVB version.  
+![image](https://github.com/AII-SDU/EDK2_SG2042/blob/separate-v2/edk2-platforms/Platform/Sophgo/SG2042Pkg/Documents/Media/Sophgo_SG2042_EVB.png)  

```

Comment:
Please check these links. I am unable to access.

Reply:
Sorry, this is a link to a private repository, we will change the link to a public repository.

Reply:
No worries. May be you can provide relative link to the image in the same directory of the branch so that it always works for any branch you raise the PR.